doc/dev/test_proxy_troubleshooting.md

```diff
@@ -39,6 +40,49 @@ containing the strings `test_delete` or `test_upload`.
 
 For more information about `pytest` invocations, refer to [Usage and Invocations][pytest_commands].
 
+## ServiceRequestError: Cannot connect to host
+
+Async tests may occasionally fail during startup with the following exception:
```

Comment:
Have you seen this self-mitigate? Is it legit a

> every once in the while

? Or is this specific to the user setup. Have you ever seen a case where the user fixes nothing and it starts working?

Reply:
I've seen this self-mitigate on my own machine, and I haven't yet found a consistent way to get around the error -- it just comes up sometimes, so I usually just target port 5000 to avoid any issues.

It's true that users need to make sure their branch is up to date with `main` in case there's a new proxy cert, but we do automatically perform the rotation if necessary at proxy startup: https://github.com/Azure/azure-sdk-for-python/blob/689bac08962828622a4262fbfecbbf5b429cb481/tools/azure-sdk-tools/devtools_testutils/proxy_startup.py#L197-L208

Reply:
Updated to recommend pulling the latest `main` before trying the workaround